src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentTypeFinder.cs

```diff
@@ -566,16 +566,13 @@ internal static class DependentTypeFinder
                 foreach (var task in tasks)
                 {
                     var result = await task.ConfigureAwait(false);
-                    if (result != null)
+                    foreach (var derivedType in result)
```

Comment:
Check for default?

(See https://github.com/dotnet/corefx/blob/master/src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray_1.cs#L1037)